cmd/helm/load_plugins.go

```diff
@@ -313,7 +313,7 @@ func loadFile(path string) (*pluginCommand, error) {
 	cmds := new(pluginCommand)
 	b, err := ioutil.ReadFile(path)
 	if err != nil {
-		return cmds, errors.New(fmt.Sprintf("File (%s) not provided by plugin. No plugin auto-completion possible.", path))
+		return cmds, fmt.Errorf("file (%s) not provided by plugin. No plugin auto-completion possible", path)
```

Comment:
should the "F" be capitalized to match the old error message or was that a linter error too?

Reply:
FYI: ```cmd/helm/load_plugins.go:316:27: error-strings: error strings should not be capitalized or end with punctuation or a newline (revive)
		return cmds, fmt.Errorf("File (%s) not provided by plugin. No plugin auto-completion possible.", path)```